bundle/manifests/gatekeeper-operator.clusterserviceversion.yaml

```
@@ -370,25 +370,6 @@ spec:
                 control-plane: gatekeeper-operator-controller-manager
             spec:
               containers:
-              - args:
-                - --secure-listen-address=0.0.0.0:8443
-                - --upstream=http://127.0.0.1:8080/
-                - --logtostderr=true
-                - --v=10
-                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
-                name: kube-rbac-proxy
-                ports:
-                - containerPort: 8443
-                  name: https
-                  protocol: TCP
-                resources: {}
-                securityContext:
-                  allowPrivilegeEscalation: false
-                  capabilities:
-                    drop:
-                    - ALL
-                  seccompProfile:
-                    type: RuntimeDefault
               - args:
                 - --health-probe-bind-address=:8081
                 - --metrics-bind-address=127.0.0.1:8080
```

We need to not make this `127.0.0.1` so that Prometheus can reach the metrics endpoint. I think this should be okay:
```suggestion
                - --metrics-bind-address=:8080
```

---

An artifact of me looking into replacing it with `controller-runtime`. It's updated. 🙂